spec/app_support/reservations/duration_change_validations_spec.rb

```diff
@@ -63,16 +61,32 @@
             before { allow(reservation).to receive(:in_cart?).and_return(false) }
 
             it "does not allow changing the start time" do
+              reservation.reserve_start_at -= 2.hours
+              reservation.reserve_end_at -= 2.hours
+
               expect(validator).not_to be_valid
               expect(validator.errors.full_messages)
                 .to include("Reserve start at cannot change once the reservation has started")
             end
+
+            it "is allowed to extend the reservation" do
+              reservation.reserve_end_at += 10.minutes
+              expect(validator).to be_valid
+              expect(validator.errors).to be_empty
+            end
+
+            it "is not allowed to shorten the reservation" do
+              reservation.reserve_end_at -= 5.minutes
+              expect(validator).not_to be_valid
+              expect(validator.errors.full_messages)
+                .to include("Reserve end at cannot be shortened inside the lock window")
```

Comment:
I'm wondering about the wording of the error messages here.  Would 'reservation end time cannot be shortened...' be clearer than 'reserve end at cannot be shortened...' ?

Reply:
@jhanggi I'm not sure what the history is behind the human-readable version of `reserve_end_at` is. Idk if it's worth changing though (i.e. would it cause more user confusion to change now).